javav2/example_code/ec2/src/main/java/com/example/ec2/DescribeInstances.java

```diff
@@ -25,37 +21,33 @@
  * https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/get-started.html
  */
 public class DescribeInstances {
-    // private static Global logger.
-    private Logger logger = (Logger) LoggerFactory.getLogger(this.getClass());
-
     public static void main(String[] args) {
         Region region = Region.US_EAST_1;
         Ec2Client ec2 = Ec2Client.builder()
-                .region(region)
-                .build();
+            .region(region)
+            .build();
 
         describeEC2Instances(ec2);
         ec2.close();
     }
 
     public static void describeEC2Instances(Ec2Client ec2) {
-        String nextToken = null;
         try {
-            do {
-                DescribeInstancesRequest request = DescribeInstancesRequest.builder().maxResults(6).nextToken(nextToken)
-                        .build();
-                DescribeInstancesResponse response = ec2.describeInstances(request);
-                for (Reservation reservation : response.reservations()) {
-                    for (Instance instance : reservation.instances()) {
-                        System.out.println("Instance Id is " + instance.instanceId());
-                        System.out.println("Image id is " + instance.imageId());
-                        System.out.println("Instance type is " + instance.instanceType());
-                        System.out.println("Instance state name is " + instance.state().name());
-                        System.out.println("monitoring information is " + instance.monitoring().state());
-                    }
-                }
-                nextToken = response.nextToken();
-            } while (nextToken != null);
+            DescribeInstancesRequest request = DescribeInstancesRequest.builder()
+                .maxResults(10)

```

Comment:
Do you need to set maxResults here? You might have done this to test it.

Reply:
Just a default value.